addons/website/static/tests/tours/page_manager.js

```diff
@@ -0,0 +1,63 @@
+/** @odoo-module **/
+
+import wTourUtils from 'website.tour_utils';
+
+const deleteSelectedPage = [
+    {
+        content: "Pick Action",
+        trigger: '.o_cp_action_menus button',
+        run: 'click',
+    },
+    {
+        content: "Pick Delete",
+        trigger: '.o-dropdown--menu span:contains("Delete")',
+        run: 'click',
+    },
+    {
+        content: "Pick I am sure about this",
+        trigger: 'main.modal-body input[type="checkbox"]',
+        run: 'click',
+    },
+    {
+        content: "Pick OK",
+        trigger: '.modal-content footer button.btn-primary',
+        run: 'click',
+}];
+const homePage = 'tr:contains("Home"):has(td[name="website_id"]:empty)';
+wTourUtils.registerWebsitePreviewTour('website_page_manager', {
```

Comment:
Add a blank line before your `registerWebsitePreviewTour` it will ease reading the file